ecommerce/serializers.py

```diff
@@ -367,15 +367,26 @@ def _update_items(cls, basket, items):
             item = items[0]
             product_id = item.get("product_id")
             run_ids = item.get("run_ids")
-            product = models.Product.objects.get(id=product_id)
+            try:
+                # product_id = product_id
+                product_id = int(product_id)
```

Comment:
Instead of raising `ValueError` below, why not utilize python `str.isdigit()` here ?

```
if product_id and product_id.isdigit():
     ....
 
```